### DISCUSSION TITLE
Is there a way to fetch metadata links (specifically Deezer) from musicbrainz isntead of Lidarr?

### DISCUSSION THREAD

**Discussion by rairulyle** [RandomNinjaAtk/arr-scripts#152] [Q&A] [answered]
I have this issue where the Deezer link is missing in musicbrainz and I added as per instruction in the logs. The problem is Lidarr gets it after Refresh & Scan but does not save it to DB (Link appears in UI and then disappears). Therefore, the deezer link is not fetched by the script.

I know that this is an issue by Lidarr but I hope that we can have like a fallback when it is not existing?

BEFORE SCAN
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/12708526/698f1eff-fb30-4433-a818-ab5840fcb158)

AFTER SCAN (without refreshing the page)
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/12708526/dff956a6-a2a3-40d0-bb87-dc5b596010ed)

AFTER SCAN (refreshing the page)
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/12708526/f4bffe73-0b0c-44af-9a3a-da19ad3387c4)

**Comment by RandomNinjaAtk** [comment, marked as answer]
No, the script used to do this but it was overwhelming the website... So the feature had to be removed...

If this ever gets implemented, it would solve it: https://github.com/Lidarr/Lidarr/issues/1089

The links should populate for artists in Lidarr, but not albums, which is what is actually needed at this point. As far as URLs disappearing/re-appearing on refreshes, might be worth contacting Lidarr about that issue...
